EIPS/eip-draft_transaction_representation.md

```diff
@@ -0,0 +1,67 @@
+---
+eip: 7023
+title: Automatic Transaction Representation v1
```

Comment:
We don't normally put versioning in the title. Instead, you can use the EIP number when referring to a particular version of transaction representation.

```suggestion
title: Automatic Transaction Representation
```